test/Motor.Extensions.Utilities_IntegrationTest/DemonstrationTests.cs

```diff
@@ -103,9 +102,6 @@ protected class ReverseStringConverter : ISingleOutputService<string, string>
             public Task<MotorCloudEvent<string>?> ConvertMessageAsync(MotorCloudEvent<string> dataCloudEvent,
                 CancellationToken token = default)
             {
-                var parentContext =
-                    dataCloudEvent.Extension<DistributedTracingExtension>()?.GetActivityContext();
```

Comment:
Why did you removed this check?

Reply:
`parentContext` cannot be `null` so the assert below doesn't make much sense and without the assert, what's the point of this line? If we want to check the context here somehow, then we need a better assertion.

Reply:
It would great to check if you get a correct context.

Reply:
We already did that. The check compares the `parentContext` to `default` now.